src/mavis/convert/vcf.py

```diff
@@ -192,6 +237,20 @@ def convert_record(record: VcfRecordType) -> List[Dict]:
                     COLUMNS.break2_position_end: end,
                 }
             )
+        elif (
+            info.get('CILEN') != None
```

Comment:
you should  use `is` or `is not ` for comparison with None/True/False, that said, here the comparator is unecessary and it would be more pythonic to just use the truthyness

```python
elif info.get('CILEN'):
```